shared/src/sector/registered_proof.rs

```diff
@@ -25,6 +26,12 @@ pub enum RegisteredSealProof {
     StackedDRG8MiBV1P1,
     StackedDRG32GiBV1P1,
     StackedDRG64GiBV1P1,
+
+    StackedDRG2KiBV1P1_Feat_SyntheticPoRep,
```

Comment:
Ok.... why `_Feat_SyntheticPoRep`. Can we not just assign a new version?

Like.... we had V1. Then we tacked on a P1 instead of bumping it to V2. Now we're adding `_Feat_SyntheticPoRep` instead of just bumping it to P2 or V2 or something???

Reply:
The PoRep proof version isn't changing, it's an optional feature that's been added -- and that's the naming convention we chose for optional features that are enabled by new proof types.

Reply:
Where do we actually use this? In the builtin actors, it looks like we don't care.

Reply:
Hm. Let's continue on slack and schedule a time tomorrow to talk about this. I think we can land this PR quickly, but doing this async will take too long.